Sofascore/Sofascore/Networking/ApiClient.swift

```diff
@@ -0,0 +1,61 @@
+import Alamofire
+import Foundation
+
+struct ApiClient {
+    
+    let urlString = "https://static-api.sofascore.dev/api/event/11352380"
```

Comment:
Ovo funkcionira kao primjer s jednim pozivom, ali probaj to sagledati na nacin da trebas recimo informacije o eventu s drugim id-jem? Dobro je razmisljati o takvim stvarima unaprijed i ovo je slucaj gdje mozemo biti sigurni da ce se tako nesto i dogoditi. Taj parametar bi svakako trebalo prosljedivati izvana. 

Ono sto si jos mozes pogledati je struktura api url-a (hint: vidi base url, path)